clientv3/balancer.go

```diff
@@ -45,7 +45,9 @@ type balancer interface {
 	// pinned returns the current pinned endpoint.
 	pinned() string
 	// endpointError handles error from server-side.
-	endpointError(addr string, err error)
+	// returns true when addr is marked as unhealthy.
+	// returns false when addr was stale endpoint.
```

Comment:
what is a stale endpoint? this func should do only one thing as reporting endpoint error, not checking if the endpoint is stale.